src/main/webapp/util.js

```diff
@@ -103,15 +103,15 @@ function xhrDelete(url, callback, errback){
 }
 
 function parseJson(str){
-	return JSON.parse(str);
+	return window.JSON ? JSON.parse(str) : eval('(' + str + ')');
```

Comment:
Evaluating untrusted strings as JavaScript code just seems like a bad idea, any reason to change this back?